test/testutil.c

```diff
@@ -185,18 +185,20 @@ static void test_fail_message(const char *prefix, const char *file, int line,
 static void test_fail_message_va(const char *prefix, const char *file, int line,
                                  const char *type, const char *fmt, va_list ap)
 {
-    fputs(prefix != NULL ? prefix : "ERROR", stderr);
-    fputs(":", stderr);
+    BIO *b_stderr = BIO_new_fp(stderr, BIO_NOCLOSE | BIO_FP_TEXT);
```

Comment:
Agreed it's silly. I was trying to minimize the number of changes; and given this is for test, it's likely that this might only be called once.